### DISCUSSION TITLE
"Discharge capacity [A.h]" after simulate Coupled degradation mechanisms is bigger than "Nominal cell capacity [A.h]"

### DISCUSSION THREAD

**Discussion by 1028fgh** [pybamm-team/PyBaMM#4306] [Q&A]
Hello,everyone

[# Modelling coupled degradation mechanisms in PyBaMM](https://docs.pybamm.org/en/stable/source/examples/notebooks/models/coupled-degradation.html)

I used the model settings from the website above to conduct multi-stage charging experiment simulations. PyBaMM version: 24.5
Unfortunately, the simulation results showed that **"Discharge capacity [A.h]"** would be bigger than **"Nominal cell capacity [A.h]"**, which is unreasonable in the real world.

Since I don't fully understand the material engineering field, I would like to ask if there is a bug in the settings in ```options```. Thank you very much.

```python
import pybamm
import numpy as np
options ={
        "cell geometry": "arbitrary",
        "thermal": "lumped",
                
        "SEI": "ec reaction limited",
        "SEI porosity change": "true",
        "SEI on cracks": "true",
        "lithium plating": "partially reversible",
        "lithium plating porosity change": "true",  # alias for "SEI porosity change"
        "particle mechanics": ("swelling and cracking", "swelling only"),
        "loss of active material": "stress-driven",
        "calculate discharge energy": "true",  # for compatibility with older PyBaMM versions
}
ParameterValue = pybamm.ParameterValues("OKane2022")
model = pybamm.lithium_ion.DFN(options=options, )
var_pts = {
        "x_n": 5,  # negative electrode
        "x_s": 5,  # separator
        "x_p": 5,  # positive electrode
        "r_n": 30,  # negative particle
        "r_p": 30,  # positive particle
}

pybamm.set_logging_level("NOTICE")
Current_values_list = [2.510, 2.405, 2.308, 2.239, 2.186,]
CC_experiment_list = [f"Charge at {current_values}A until 4.2 V" for current_values in Current_values_list]
experiments = pybamm.Experiment([tuple(CC_experiment_list), "Hold at 4.2V until 50 mA"] * 1)
initial_soc = 0
casadi_solver = pybamm.CasadiSolver(root_method='lm', dt_max=1)
sim = pybamm.Simulation(model, experiment=experiments, parameter_values=ParameterValue, solver=casadi_solver)
sol = sim.solve(solver=casadi_solver, initial_soc=initial_soc, showprogress='true')

maximum_available_capacity = np.abs(sol["Discharge capacity [A.h]"].entries[-1])
print(maximum_available_capacity)
nominal_cell_capacity = ParameterValue["Nominal cell capacity [A.h]"]
print(nominal_cell_capacity)

```

**Comment by pkalbhor** [comment]
The nominal cell capacity is primarily used for calculating the C-rate and has limited application in other contexts. If you need to determine the actual capacity, it's recommended to use the `Cyclable Lithium Capacity [A.h]`. You can obtain this value with the following command:
```python
sol.summary_variables['Cyclable lithium capacity [A.h]']
```
I hope this information is helpful!

**Reply by 1028fgh** [reply]
Hello!
I'd love to know how you managed to successfully call ```['Cyclable lithium capacity [A.h]']``` or ```sol.summary_variables```. 
After I changed the above ```"Nominal cell capacity [A.h]"``` to ```"Cyclable lithium capacity [A.h]"```, and I got the following error message:

```py
KeyError: "'Cyclable lithium capacity [A.h]' not found. Best matches are ['Nominal cell capacity [A.h]', 'Typical plated lithium concentration [mol.m-3]', 'Dead lithium decay rate [s-1]']"
```
